docs/b2b-edition/graphql/company-account-registration.mdx

```diff
@@ -0,0 +1,329 @@
+# GraphQL Storefront API: Company Account Registration
+
+B2B Edition’s GraphQL Storefront API powers the registration workflow for new Company accounts in the [Buyer Portal experience](https://support.bigcommerce.com/s/article/Buyer-Portal-Migration). The built-in form for Company account registration uses GraphQL Storefront API queries and mutations to:
+
+* Retrieve the form fields configured in your B2B Edition [account form field settings](https://support.bigcommerce.com/s/article/B2B-Edition-Settings).  
+* Check if an applicant has an existing B2C customer account and populate their information in the form.  
+* Create a pending Company account with the provided information.  
+* Generate a B2C customer account and corresponding Company user account for applicants without an existing account.
+
+While B2B Edition’s [legacy Stencil storefront experience](https://developer.bigcommerce.com/b2b-edition/docs/stencil) uses the [REST Storefront API](https://developer.bigcommerce.com/b2b-edition/apis#storefront-apis) in its registration form, it is recommended to use the GraphQL Storefront API to customize the Buyer Portal form. Additionally, the `registerCompany` mutation in [BigCommerce’s GraphQL Storefront API](https://developer.bigcommerce.com/b2b-edition/docs/account-registration) also supports Company registration for storefront users with an existing B2C customer account.
+
+## Tokens
+
+Guest users do not have an existing customer account with login credentials. As a result, the requests involved in the [guest registration workflow](#registration-for-guest-users) can be performed anonymously without a token in the Authorization header. 
+
+The [registration workflow for existing customers](#registration-for-existing-storefront-customers) passes a storefront authToken to the Authorization header. This is a Bearer token generated via the `login` mutation following customer login. While it is included in the header of all B2B Edition GraphQL queries and mutations in the flow, it is only required for the `customer` query, which is a proxy request to the [BigCommerce GraphQL Storefront API](https://developer.bigcommerce.com/docs/storefront/graphql). This request uses the following endpoint: `https://api-b2b.bigcommerce.com/api/v3/proxy/bc-storefront/graphql`
```

Comment:
The phrasing "it is only required for the `customer` query" made it sound to me like you meant that's the only query in all of the B2B GraphQL API that requires it.  (There are, of course, many queries and mutations in the B2B GraphQL that require an auth token.) I think this actually meant it's the only query in _this flow_ that requires it, but some re-wording might avoid confusion.